jsapp/js/components/processing/analysis/responseForms/responseWrapper.component.tsx

```diff
@@ -22,6 +22,8 @@ import type { AnalysisQuestionInternal } from '../constants'
 
 interface ResponseWrapperProps {
   uuid: string
+  /** Adds a clear button with the given logic */
+  clearable?: Function
```

Comment:
`clearable` sounds like a boolean. Better name it as `onClear` to sound more like a function and also to align with Mantine naming.